fdbserver/storageserver.actor.cpp

```diff
@@ -1728,6 +1728,55 @@ struct StorageServer : public IStorageMetricsService {
 		StorageBytes sb = storage.getStorageBytes();
 		metrics.getStorageMetrics(req, sb, counters.bytesInput.getRate(), versionLag, lastUpdate);
 	}
+
+	void getSplitMetrics(const SplitMetricsRequest& req) override { this->metrics.splitMetrics(req); }
+
+	void getHotRangeMetrics(const ReadHotSubRangeRequest& req) override { this->metrics.getReadHotRanges(req); }
+
+	// Used for recording shard assignment history for auditStorage
+	std::vector<std::pair<Version, KeyRange>> shardAssignmentHistory;
+	Version trackShardAssignmentMinVersion; // == invalidVersion means tracking stopped
+
+	std::string printShardAssignmentHistory() {
```

Comment:
why do you have this changes? seems unrelated to your PR?

Reply:
Thanks, these changes were required in the merge to Main branch, but not needed for release 71.3 branch. I have removed them,